src/app/app.component.css

```diff
@@ -0,0 +1,47 @@
+.main {
+  width: 500px;
+  text-align: center;
+  margin: 0 auto;
+  border: 2px solid, #d7d7d7;
+  border-bottom: 0px;
```

Comment:
Jeśli gdzieś zapisujesz zero pikseli to nie dodawaj `px` - wystarcza samo 0.
Sprawdź wszędzie

Reply:
https://github.com/jedi800/Checklistaa/pull/1/commits/6a884581fbec95456cb69bcce6f136f785c5a02a